app/routes/index.tsx

```diff
@@ -1,6 +1,18 @@
-import { Heading, Text } from "@chakra-ui/react";
+import { Heading, Text, Button } from "@chakra-ui/react";
+import { json } from "@remix-run/node";
+import type { LoaderArgs } from "@remix-run/node";
```

Comment:
Move your type imports below your regular imports and separate with a blank line:

```ts
import {...} from "..."
import {...} from "..."

import type {...} from "..."
import type {...} from "..."
```